hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -0,0 +1,200 @@
+
+[id="hcp-4-17-tenancy-aws_{context}"]
+=== Support for tenancy on {aws-short}
+When you create a hosted cluster on {aws-short}, you can indicate whether the EC2 instance should run on shared or single-tenant hardware. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-aws-deploy-hc_hcp-deploy-aws[Creating a hosted cluster on {aws-short}].
+
+[id="hcp-4-17-_{context}"]
+=== Support for {product-title} versions in hosted clusters
+You can deploy a range of supported {product-title} versions in a hosted cluster. For more information, see xref:../hosted_control_planes/hcp-updating.adoc#hcp-supported-ocp-versions_hcp-updating[Supported {product-title} versions in a hosted cluster].
+
+[id="hcp-4-17-virt-dc_{context}"]
+=== {hcp-capital} on {VirtProductName} in a disconnected environment is Generally Available
+In this release, {hcp} on {VirtProductName} in a disconnected environment is Generally Available. For more information, see xref:../hosted_control_planes/hcp-disconnected/hcp-deploy-dc-virt.adoc#hcp-deploy-dc-virt[Deploying {hcp} on {VirtProductName} in a disconnected environment].
+
+[id="hcp-4-17-multi_{context}"]
+=== {hcp-capital} for an ARM64 {product-title} cluster on {aws-short} is Generally Available
+In this release, {hcp} for an ARM64 {product-title} cluster on {aws-short} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-enable-arm-amd_hcp-deploy-aws[Running hosted clusters on an ARM64 architecture].
+
+[id="hcp-4-17-aws_{context}"]
+=== {hcp-capital} for an {product-title} cluster on {aws-short} is Generally Available
+In this release, {hcp} for an {product-title} cluster on {aws-short} is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-enable-arm-amd_hcp-deploy-aws[Deploying hosted clusters on an {aws-short} architecture].
+
+[id="hcp-4-17-non-bm_{context}"]
+=== {hcp-capital} for an {product-title} cluster on non-bare-metal agent machines is Generally Available
+In this release, {hcp} for an {product-title} cluster on non-bare-metal agent machines is Generally Available. For more information, see xref:../hosted_control_planes/hcp-deploy/hcp-deploy-non-bm.adoc#hcp-deploy-non-bm[Deploying hosted clusters on non-bare-metal agent machines].
+
+[id="hcp-4-17-bm_{context}"]
+=== {hcp-capital} for an {product-title} cluster on bare metal is Generally Available
+In this release, {hcp} for an {product-title} cluster on bare metal is Generally Available. For more information, see xref:../hosted_control_planes/hosted_control_planes/hcp-manage/hcp-manage-bm.adoc[Running hosted clusters on bare metal].
+
+[id="hcp-4-17-virt_{context}"]
```

Comment:
@xenolinux I think lines 54 - 56 should be removed, since HCP on OpenShift Virtualization has been GA for several releases now.

Reply:
Addressed